### PR TITLE
feat(judicial-system): restriction request pdf strings in contentful

### PR DESCRIPTION
**Pull request by VilliGunn**
# Use Contentful for restriction request PDF

https://app.asana.com/0/1199153462262248/1200550897975596

## What

- Use strings from Contentful in restriction request PDF
- Added a new namespace `judicial.system.backend`
- Added the `extract-strings` for judicial-system-backend to workspace
- Updated helm charts

## Why

We want to be able to control text in backend through Contentful

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/backend/src/app/modules/case/case.service.ts [hunk 1]
```diff
@@ -309,12 +311,12 @@ export class CaseService {
...
   }
 
-  getRequestPdf(existingCase: Case): Promise<string> {
+  async getRequestPdf(existingCase: Case): Promise<string> {
     this.logger.debug(
       `Getting the request for case ${existingCase.id} as a pdf document`,
     )
-
-    return getRequestPdfAsString(existingCase)
+    const intl = await this.intlService.useIntl(['judicial.system.test'], 'is')
```
Comment (VilliGunn) [inline apps/judicial-system/backend/src/app/modules/case/case.service.ts:318]: Leftover from testing
```suggestion
    const intl = await this.intlService.useIntl(['judicial.system.backend'], 'is')
```

File: workspace.json [hunk 2]
```diff
@@ -5249,7 +5249,13 @@
             "cwd": "apps/judicial-system/backend"
           }
         },
-        "docker-express": {}
+        "docker-express": {},
+        "extract-strings": {
+          "builder": "@nrwl/workspace:run-commands",
+          "options": {
+            "command": "yarn ts-node -P libs/localization/tsconfig.lib.json libs/localization/scripts/extract 'apps/judicial-system/backend/src/app/messages/**/*.{js,ts,tsx}'"
```
Comment (jeremybarbet) [inline workspace.json:5256]: I guess you can make this path stricter to either target the `requestPdf.ts` directly or just the `*.ts` within the folder

Comment (VilliGunn) [reply workspace.json:5256]: Yes, definitely! I'll remove the js and tsx but we plan on having more translations on the messages folder so I want to target the whole folder 😃

**Review by jeremybarbet — APPROVED** [review]
LGTM

**Review by oddsson — APPROVED** [review]

**Review by eirikurn — CHANGES_REQUESTED** [review]

**Review by pshomov — APPROVED** [review]
LGTM

**Review by eirikurn — APPROVED** [review]
Thanks!

**Pull request merged**